test/resources/base.py

```diff
@@ -27,33 +24,25 @@ def __init__(self):
         super(BaseResource, self).__init__()
         self.api_request = request
         self.req_obj = HttpRequestFactory.create('requests')
-        """
-        @ICK: try moving these headers to the config file!
-        """
-        headers = {
-            'content-type': 'application/json',
-            'X-Yummly-App-ID': '9cce27e7',
-            'X-Yummly-App-Key': 'b13c741344519e5f89cb0edb7e8043f6'
-        }
         self.req_obj.set_headers(headers)
```

Comment:
`global name 'headers' is not defined.` use CFG_OBJ.get(<variable name here from config>)
you cannot take out the variable instantiation and continue to use the variable and make references to a non existing variable.